templates/squad/show.html.twig

```diff
@@ -4,31 +4,18 @@
 	<div class="container">
 		<h1>Mes Squads</h1>
 		<div class="row">
-			{% for squad in squad %}
+			{% for item in squad %}
```

Comment:
Il aurait été mieux d'avoir nommer squad au pluriels dans ton controller afin d'avoir quelque chose de mieux 
`{% for squad in squads %}`